common/lib/client/realtimechannel.ts

```diff
@@ -643,73 +675,74 @@ var RealtimeChannel = (function() {
 			case 'attached':
 				/* resume any sync operation that was in progress */
 				this.sync();
+				break;
 			default:
```

Comment:
I'm on the fence about this one. It's a bit more expressive to keep this here since it explicitly communicates to the reader that we don't want to handle any of the other cases. I'll probably leave it in for now and later on we can add an ESLint rule that enforces a standard for this situation.